src/components/beatmap/PlaylistThumbnails.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <v-container :class="'pa-0 pl-3'">
+    <v-tooltip v-for="value in playlists" :key="value.key" top>
+      <template #activator="{ on }">
+        <v-chip :v-on="on" color="transparent">
+          <PlaylistCoverAvatar
+            :playlist="value"
+            :avatar-size="24"
+            :icon-expand-size="16"
+          />
+        </v-chip>
```

Comment:
The reason the tooltip wasn't show is because `PlaylistCoverAvatar` already has a `v-hover` in and capture the event. As for v-chip I'm not sure for the reason of it. I'm suggesting this instead which would work just fine imo:

```vue
<v-avatar :size="24" class="mx-1" v-on="on">
  <PlaylistCover :playlist="playlist" />
</v-avatar>
```

Reply:
v-chip was chosen because of insufficient knowledge of available vue components ^^ thanks for the info